client/src/raw.rs

```diff
@@ -262,6 +262,15 @@ where
             r.extensions_mut().insert(labels);
         }
     );
+    proxy!(
+        get_workflow_execution_history_reverse,
+        GetWorkflowExecutionHistoryReverseRequest,
+        GetWorkflowExecutionHistoryReverseResponse,
+        |r| {
+            let labels = AttachMetricLabels::namespace(r.get_ref().namespace.clone());
+            r.extensions_mut().insert(labels);
+        }
+    );
```

Comment:
Is there an easy way to make a test or compile fail if one of these is missing from the generated `WorkflowService`?